### PR TITLE
[minigraph.py]: Parse peer switch info from minigraph

### PR DESCRIPTION
**Pull request by theasianpianist**
**- Why I did it**

**- How I did it**
Add `PEER_SWITCH` table to config_DB:

```
PEER_SWITCH: {
	switch_name: {
		address_ipv4: {
			'ipv4 address'
		}
	}
}
```    

Add dual ToR subtype to config_DB:

```
DEVICE_METADATA: {
	localhost: {
		...
		subtype: 'DualToR'
		...
	}
}
```

**- How to verify it**

**- Which release branch to backport (provide reason below if selected)**

- [ ] 201811
- [ ] 201911
- [ ] 202006

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by theasianpianist** [comment]
retest this please

**Comment by theasianpianist** [comment]
retest vs please

File: src/sonic-config-engine/minigraph.py [hunk 1]
```diff
@@ -674,16 +686,31 @@ def parse_linkmeta(meta, hname):
...
+        upper_tor_hostname = ''
+        lower_tor_hostname = ''
+
         properties = linkmeta.find(str(QName(ns1, "Properties")))
         for device_property in properties.findall(str(QName(ns1, "DeviceProperty"))):
             name = device_property.find(str(QName(ns1, "Name"))).text
             value = device_property.find(str(QName(ns1, "Value"))).text
             if name == "FECDisabled":
                 fec_disabled = value
+            if name == "GeminiPeeringLink":
```
Comment (prsunny) [inline src/sonic-config-engine/minigraph.py:699]: can we have it as "else if"?

**Comment by theasianpianist** [comment]
retest vs please

**Review by prsunny — APPROVED** [review]

**Pull request merged**